src/ooxml/testcases/org/apache/poi/xssf/usermodel/TestXSSFName.java

```diff
@@ -130,4 +132,22 @@ public void testSetNameName() throws Exception {
 
         wb.close();
     }
+
+    @Test
+    public void testSetNameNameLongName() throws IOException {
+        // Test that renaming named ranges doesn't break our new named range map
+        XSSFWorkbook wb = new XSSFWorkbook();
+        wb.createSheet("First Sheet");
+
+        //Name with very long name. Looks like a cell reference but this is not a reference.
+        XSSFName nameSheet1 = wb.createName();
+        nameSheet1.setNameName("F04030020010");
```

Comment:
You mean to change the code inside XSSFName.setNameName and not altering the test case?

Sorry for my question but your comment is just below the test case, so I'm a little bit confused.